src/summariser.py

```diff
@@ -2,17 +2,48 @@
 from cohere.types.non_streamed_chat_response import NonStreamedChatResponse
 from src.utils import get_cohere_api_key
 import logging
+from langchain_community.document_loaders import WebBaseLoader
+import re
+from langchain_core.documents.base import Document
 
 logger = logging.getLogger(__name__)
 
 
+def extract_urls(text):
+    url_pattern = re.compile(r"(https?://\S+|www\.\S+)")
+    urls = url_pattern.findall(text)
+    return urls
+
+
 def read_prompt_header() -> str:
     # Read the prompt from the file
     with open("src/prompt.txt", "r") as file:
```

Comment:
This is hardcoded, I'd rather have a function that takes a prompt path as an input.